components/plan-build/bin/public.bash

```diff
@@ -588,6 +597,7 @@ update_pkg_version() {
 __resolve_version_placeholder(){
     local original=${1}
     local real_version=${2}
+    # shellcheck disable=2001
```

Comment:
This is equivalent, but is it more readable?
```bash
echo "${original//__pkg__version__unset__/${real_version}}"
```

Reply:
Less readable, imo.

Reply:
Noted. I'll leave it be.